udata_front/theme/gouvfr/assets/js/components/MarkAsNoSpam/MarkAsNoSpam.stories.ts

```diff
@@ -0,0 +1,27 @@
+import MarkAsNoSpam, { type MarkAsNoSpamProps } from './MarkAsNoSpam.vue';
+import type { Meta, StoryObj } from "@storybook/vue3";
+
+const meta = {
+  title: 'Components/MarkAsNoSpam MarkAsNoSpam',
+  component: MarkAsNoSpam,
+} satisfies Meta<typeof MarkAsNoSpam>;
+
+export default meta;
+
+const args: MarkAsNoSpamProps = {
```

Comment:
`MarkAsNoSpam` is not defined in MarkAsNoSpam.vue. You can extract the props to make a type like this : https://github.com/etalab/udata-front/blob/master/udata_front/theme/gouvfr/assets/js/components/dataset/CardMD.vue#L68

Reply:
Done! I'm used to use the `InstanceType` in my own projects:
```
type MarkAsNoSpamProps = InstanceType<typeof MarkAsNoSpam>["$props"];
```
It's a little bit less verbose (no additional `script` tag with an export) but it may be less readable…